test/boost/transport_test.cc

```diff
@@ -83,11 +83,11 @@ SEASTAR_THREAD_TEST_CASE(test_response_request_reader) {
 
     using sc = cql_transport::event::schema_change;
     res.serialize({sc::change_type::CREATED, sc::target_type::KEYSPACE, "foo"}, version);
-    res.serialize({sc::change_type::CREATED, sc::target_type::TABLET_KEYSPACE, "foo"}, version);
     res.serialize({sc::change_type::CREATED, sc::target_type::TABLE, "foo", "bar"}, version);
     res.serialize({sc::change_type::CREATED, sc::target_type::TYPE, "foo", "bar"}, version);
     res.serialize({sc::change_type::CREATED, sc::target_type::FUNCTION, "foo", "bar", "zed"}, version);
     res.serialize({sc::change_type::CREATED, sc::target_type::AGGREGATE, "foo", "bar", "zed"}, version);
+    res.serialize({sc::change_type::CREATED, sc::target_type::TABLET_KEYSPACE, "foo"}, version);
```

Comment:
why this change in this patch?
doesn't it belong to the patch that introduced it?

Reply:
It's in the correct patch now, thanks.